core/field_colour.js

```diff
@@ -314,26 +453,51 @@ Blockly.FieldColour.prototype.dropdownCreate_ = function() {
   // Create the palette.
   var table = document.createElement('table');
   table.className = 'blocklyColourTable';
+  table.tabIndex = "0";
+  Blockly.utils.aria.setRole(table,
```

Comment:
Could we maybe add a label to the table? I think currently it just starts to list off all of the values in the cells, which might be confusing.

Reply:
We should definitely add a label. Since that requires the addition of new localized messages, and since all fields are going to need a similar change. I'll leave that for a separate PR.